hydrolib/core/io/mdu/models.py

```diff
@@ -502,12 +503,169 @@ def is_intermediate_link(self) -> bool:
         return True
 
 
+class Calibration(INIBasedModel):
+    """
+    The `[Calibration]` section in an MDU file.
+
+    This model is typically referenced under [FMModel][hydrolib.core.io.mdu.models.FMModel]`.calibration`.
+
+    All lowercased attributes match with the [Calibration] input as described in
+    [UM Sec.A.3](https://content.oss.deltares.nl/delft3d/manuals/D-Flow_FM_User_Manual_1D2D.pdf#section.A.3).
+    """
+
```

Comment:
Nowhere we have any comments in this FMModel components. Maybe let's do that in a separate enhancement issue.

Reply:
Issue: #288